forks/ansi-color/ansi-color.js

```diff
@@ -0,0 +1,37 @@
+var ANSI_CODES = {
```

Comment:
`ansi-color` threw a fit when I upgraded @opentelemetry. It was a single file, so I copy/pasted it, made a local patch, and resolved to the local file.